docs/guide/acquiring_data/jsoc.rst

```diff
@@ -0,0 +1,295 @@
+---------------------------------------
+Querying and Downloading Data from JSOC
+---------------------------------------
+
+Joint Science Operations Center (JSOC) contains data products from the Solar Dynamics Observatory,
+as well as certain other missions and instruments. These data are available from the JSOC database,
+which can be directly accessed by the online JSOC interface <http://jsoc.stanford.edu/ajax/lookdata.html>
+
+SunPy's JSOC Client provides an easier interface to query for JSOC data and make export requests.
+It uses drms module <https://github.com/kbg/drms> as its backend, and exposes a similar API as
+the VSO Client.
+
+There are two ways of downloading JSOC data. One way is using Sunpy's unified search interface,
+known as FIDO. FIDO supplies a single, easy and consistent way to to obtain most forms of physics
+data. An alternative way to fetch data from JSOC is by using the underlying JSOC Client. This option
+can be preferred when the complex searches are to be made, or when you need to separate the staging
+and downloading steps, which is not supported by FIDO.
+
+The JSOC stages data before you can download it,
+so a JSOC query is a three stage process, first you query the JSOC for records,
+a table of these records is returned. Then you can request these records to be
+staged for download and then you can download them. Fido combines the stages into 2, ``search()``
+and ``fetch()``.
+
+Setup
+-----
+
+SunPy's FIDO module is in ``sunpy.net``.  It can be imported as follows:
+
+    >>> from sunpy.net import Fido, attrs as a
+
+The JSOC client handles the particulars of how the data from
+the data provider is downloaded to your computer.
+
+Querying the JSOC
+-----------------
+
+To search for data in JSOC, your query needs at minimum a series name and a primekey.
+Different primekeys are supported by different series, and you can find out the primekeys
+supported in any series by:
+
+	>>> import drms
+	>>> c = drms.Client()
+	>>> print(c.pkeys('hmi.m_720s'))
```

Comment:
Would it be worth exposing this through the JSOCClient? Something like `JSOCClient.get_prime_keys('series')`